docs/unified-test-documentation/test-matrix.md

```diff
@@ -0,0 +1,148 @@
+# Test matrix
+
+## About
+
+<!--
+Matryca testowa służy do określenia które z zetsawów i przypadków testych
+opisanych w dokumentacji testowej dedykowane są dla danej platformy
+-->
+
+## Test matrix - ASUS KGPE-D16
```

Comment:
Platform names have to be consistent across all departments and whole documentation, if that is correct name it should be changed in other places.